packages/default/scss/toolbar/_theme.scss

```diff
@@ -1,3 +1,19 @@
+$toolbar-separator-border: inherit !default;
+
+$toolbar-button-expanded-bg: null !default;
+$toolbar-button-expanded-shadow: none !default;
+$toolbar-button-hovered-opacity: .05 !default;
+$toolbar-button-active-opacity: .22 !default;
+$toolbar-button-focused-opacity: 0 !default;
+$toolbar-button-focused-outline-opacity: .08 !default;
+
+/// Toolbar button types
+/// null - default buttons type
+/// thin - flat buttons appearance in the normal state and normal buttons appearance in other states
+/// flat - flat buttons type
+/// outline - outline buttons type
```

Comment:
I suggest leaving this variable internally documented (`//` instead of `///`) for now.